src/plugin/inputmethod.cpp

```diff
@@ -755,3 +755,19 @@ void InputMethod::onPluginPathsChanged(const QStringList& pluginPaths) {
 
     d->updateLanguagesPaths();
 }
+
+void InputMethod::showSystemSettings()
+{
+    auto previous = qgetenv("QT_WAYLAND_SHELL_INTEGRATION");
+    qunsetenv("QT_WAYLAND_SHELL_INTEGRATION");
+
+    if (qgetenv("XDG_CURRENT_DESKTOP") == "KDE") {
+        QDesktopServices::openUrl(QUrl("systemsettings://kcm_virtualkeyboard"));
+    } else {
+        QDesktopServices::openUrl(QUrl("settings://system/language"));
+    }
```

Comment:
Do we have a kcm yet for configuring maliit? Should we provide instead a more generic way of configuring things that can work on all environments? As I understand, neither of these URLs will do anything useful for maliit-keyboard yet anyway.